.changeset/cyan-chefs-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
```

Comment:
Why is this marked as `major`? Was that a mistake? I think this can be a `patch` because it fixes a bug, right?

Reply:
You're right! Accepted